backend/server.ts

```diff
@@ -28,10 +29,10 @@ const storage = multer.memoryStorage(); // Store the file in memory
 const upload = multer({ storage: storage });
 
 app.get('/upload-page', (req, res) => {
-    res.sendFile(path.join(__dirname, '../Frontend/website.html'));
+    res.sendFile(path.join(__dirname, '../Frontend/testwebsite.html'));
 });
 
-app.post('/upload', upload.single('packageContent'), (req, res, next) => {
+app.post('/upload', upload.single('packageContent'), async (req, res, next) => {
     try {
         if (!req.file) {
```

Comment:
I would abstract all the S3 upload stuff into a separate function the apiPackage.ts for consistency with the other endpoints. But up to you.